src/Model/SnapUser.hs

```diff
@@ -0,0 +1,28 @@
+module Model.SnapUser where
+  
+import Data.Time.Calendar
+import Snap.Snaplet.PostgresqlSimple
+
+data SnapUser = SnapUser
+  { id :: Int
+  , name :: String
+  , email :: String
+  , birthDate :: Day
+  , password :: String
+  , genre :: String
```

Comment:
Do you mean `gender` here?
If that is the case, I think you can create a sum type to represent the possible options:
```haskell
data Gender = Female | Male | Other deriving (Show, Eq)
```
And we can create a data type to represent that in the database. Adding another migration for that:
```sql
CREATE TYPE Gender AS ENUM ('Male', 'Female', 'Other')
```
More info [here](https://www.postgresql.org/docs/9.5/static/sql-createtype.html).

Reply:
"Other" :blush:

Reply:
I don't really like enum types at the database level... from my experience they are hard to mantain and migrate... but it is a good exercise, give it a try @kinorius123